packages/flutter/lib/src/material/scrollbar.dart

```diff
@@ -174,7 +171,7 @@ class _ScrollbarState extends RawScrollbarState<Scrollbar> {
       if (states.contains(MaterialState.hovered) && widget.showTrackOnHover)
         return widget.hoverThickness ?? _kScrollbarThicknessWithTrack;
       // The default scrollbar thickness is smaller on mobile.
-      return widget.thickness ?? (_kScrollbarThickness / (_isMobile ? 2 : 1));
+      return widget.thickness ?? (_kScrollbarThickness / (_useAndroidScrollbar ? 2 : 1));
```

Comment:
This change the iPhone behavior?

Reply:
We can just remove the override `didChangeDependencies` and provide a method,
```dart
  double _getDefaultThickness() {
    final ThemeData theme = Theme.of(context);
    switch (theme.platform) {
      case TargetPlatform.android:
        return _kScrollbarThickness;
      default:
        return _kScrollbarThickness / 2;
    }
  }
```

Reply:
> This change the iPhone behavior?

yep, the `CupertinoScrollbar` override this!

Reply:
It does, but I have another change on the way to address that. Now that the refactor to RawScrollbar is complete, I am going to follow up to restore the adaptive behavior that returns the CupertinoScrollbar on iOS. :)